### PR TITLE
minor update: minor update to fix the header including in the 'source'

### PR DESCRIPTION
**Pull request by wbpcode**
Commit Message: minor update: minor update to fix the header including in the 'source'
Additional Description:

The relative path to the workspace root should be used for the header file including. But some lines are wrong and this patch fix all this type issues (?) in the 'source'.

Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: source/common/buffer/watermark_buffer.cc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 #include "source/common/buffer/watermark_buffer.h"
-#include "watermark_buffer.h"
```
Comment (nezdolik) [inline source/common/buffer/watermark_buffer.cc:2]: hm, so before in this case same header was included twice?

**Review by nezdolik — APPROVED** [review]
Looks like there is a test flake

File: source/common/buffer/watermark_buffer.cc (hunk 1 above)
Comment (wbpcode) [reply source/common/buffer/watermark_buffer.cc:2]: #pragma once will ensure the content won't be repeated

Comment (nezdolik) [reply source/common/buffer/watermark_buffer.cc:2]: right, missed that one :)

**Pull request merged**